### PR TITLE
EREGCSC-2373 -- Policy Repository/Subject Page Header

### PR DESCRIPTION
**Pull request by PhilR8**
Resolves [EREGCSC-2373](https://jiraent.cms.gov/browse/EREGCSC-2373)

**Description**
Add simple subject header to Policy Repository if subject is selected.  The formatting should match the Subject ToC formatting, but with a different font and font-size.

**This pull request changes:**

- Adds `SelectedSubjectHeading.vue` component
- Adds `getSubjectNameParts` filter method and related unit test
- Re-enables Merriweather font face, which has not been loading on `prod` for some time
- Adds related assertions in end to end test suite

**Steps to manually verify this change:**

1. Visit the Policy Repository [experimental deployment](https://tr4u7qo4f4.execute-api.us-east-1.amazonaws.com/dev1084)
2. Add and remove subjects to add and remove the Subject Header
3. Compare styling to [Figma comp](https://www.figma.com/proto/cKtR3bsPaVXFs6qfE64FZm/Usability-test?page-id=0%3A1&type=design&node-id=1-2&viewport=365%2C750%2C0.78&t=bnsZfEaxefWSiBzc-1&scaling=min-zoom)

### CODE REVIEW

File: solution/ui/regulations/.eslintrc.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
     "plugins": ["vue"],
     "rules": {
         "eqeqeq": "off",
+        "import/no-unresolved": "off",
```
Comment (PhilR8) [inline solution/ui/regulations/.eslintrc.json:19]: our vite configs need some tweaking in order for our (my) IDE to recognize aliased imports.  Until then, I'm turning this rule off.

File: solution/ui/regulations/css/scss/_application_settings.scss [hunk 2]
```diff
@@ -189,6 +189,22 @@ $policy_subject_close_background_hover: $mid_gray_3;
...
 }
 
+@font-face {
+    font-family: "Merriweather";
+    font-style: normal;
+    font-weight: 400;
+    src: url("#{$additional-font-path}/Merriweather-Regular.ttf")
+        format("truetype");
+}
+
```
Comment (PhilR8) [inline solution/ui/regulations/css/scss/_application_settings.scss:199]: We had apparently not been rendering Merriweather fonts on the site?  So this fixed that.

File: solution/ui/regulations/css/scss/_application_settings.scss [hunk 3]
```diff
@@ -189,6 +189,22 @@ $policy_subject_close_background_hover: $mid_gray_3;
...
+@font-face {
+    font-family: "Merriweather";
+    font-style: nbold;
```
Comment (PhilR8) [inline solution/ui/regulations/css/scss/_application_settings.scss:202]: fix this typo

File: solution/ui/regulations/css/scss/_application_settings.scss (hunk 2 above)
Comment (peggles2) [reply solution/ui/regulations/css/scss/_application_settings.scss:199]: nice catch

**Review by peggles2 — APPROVED** [review]

**Pull request merged**